src/main/java/com/gizasystems/automationexercise/pages/CartPage.java

```diff
@@ -13,6 +13,24 @@ public class CartPage {
     private final By productName_h4 = By.xpath("//td[@class='cart_description']//h4");
     private final By viewCart_a = By.xpath("//div[contains(@class,'confirm')]//a[@href='/view_cart']");
 
+    private String url = System.getProperty("baseUrl") + "/view_cart";
+
+    private final By firstProductName = By.cssSelector("a[href='/product_details/1']");
+
+    private final By secondProductName = By.cssSelector("a[href='/product_details/2']");
+
+    private final By firstProductPrice = By.xpath("(//td[@class='cart_price']//p)[1]");
+
+    private final By secondProductPrice = By.xpath("(//td[@class='cart_price']//p)[2]");
+
+    private final By firstProductQuantity = By.xpath("(//button[@class='disabled'])[1]");
+
+    private final By secondProductQuantity = By.xpath("(//button[@class='disabled'])[2]");
+
+    private final By firstProductTotal = By.xpath("(//p[@class=\"cart_total_price\"])[1]");
+
+    private final By secondProductTotal = By.xpath("(//p[@class=\"cart_total_price\"])[2]");
```

Comment:
No need to add empty line between locators, please remove them.
I also believe that we can enhance those locators to minimize the use of index